styleguide/_themes/global/scss/fonts.scss

```diff
@@ -22,7 +22,7 @@ $standard-line-height: 20px;
 @mixin subhead($color: $gray-darkest) {
   color: $color;
   font-family: $subhead;
-  font-weight: 600;
+  font-weight: 500;
```

Comment:
We don't want to change the font weights on anything other than solve.  So I think these references are the only ones that we need to change: 
```
styleguide/derek/templates/thought-leadership-overview/_thought-leadership.scss:  font-weight: 600;
styleguide/derek/templates/thought-leadership-overview/_thought-leadership.scss:  font-weight: 600;
styleguide/derek/templates/thought-leadership-overview/_thought-leadership.scss:  font-weight: 600;
styleguide/derek/templates/thought-leadership-overview/_thought-leadership.scss:  font-weight: 600;
styleguide/derek/templates/thought-leadership-overview/_thought-leadership.scss:  font-weight: 600;
styleguide/derek/templates/thought-leadership-overview/_thought-leadership.scss:  font-weight: 600;
styleguide/derek/templates/thought-leadership-overview/_thought-leadership.scss:  font-weight: 600;
```